server/balancer_test.go

```diff
@@ -284,44 +338,44 @@ func (s *testBalanceStorageSchedulerSuite) TestReplicas3(c *C) {
 	c.Assert(sb.cache.get(1), IsTrue)
 	sb.cache.delete(1)
 
-	// Store 7 has different zone with other stores but larger storage ratio than store 1.
-	tc.addLabelsStore(9, 1, 0.6, map[string]string{"zone": "z2", "rack": "r1", "host": "h1"})
+	// Store 9 has different zone with other stores but larger storage ratio than store 1.
+	tc.addLabelsStore(9, 9, map[string]string{"zone": "z2", "rack": "r1", "host": "h1"})
 	c.Assert(sb.Schedule(cluster), IsNil)
 }
 
-func (s *testBalanceStorageSchedulerSuite) TestReplicas5(c *C) {
+func (s *testBalanceRegionSchedulerSuite) TestReplicas5(c *C) {
```

Comment:
Some comments should be updated in this test case. 'ratio' -> 'region count' or 'region score'.